### ISSUE TITLE
Browse Components

### ISSUE THREAD

**Issue by artlowel** [DSpace/dspace-angular#111]
This task depends on #98

Add browse components to the community and collection pages.

Use the DSOListComponent to show the communities and collections on the community homepage and the items on a collection homepage

#### For all these browse components:
Hide the pager if there is only one page, also hide the gear. There should be options for this on the DSOListComponent.

#### For the items on a collection homepage: 
Put the browse component underneath the current homepage contents, [like we did in the prototype](http://ui-prototype.atmire.com/collections/81?page=1). I wouldn't put it on a separate page because the main reason a user comes to the collection homepage is to see what's inside. That shouldn't require an extra click

Sort them by issue date, most recent first by default, but as in the current xmlui, add navigation above the list to switch to "browse by title". Use bootstrap's [pills](https://v4-alpha.getbootstrap.com/components/navs/#pills) or [tabs](https://v4-alpha.getbootstrap.com/components/navs/#tabs) for that
No need for browse by author, or an alphabet list just yet.

**Comment by artlowel** [comment]
Likely relevant: https://github.com/DSpace/DSpace/pull/1775

**Comment by artlowel** [comment]
Closing, Some of these are already done, others will get separate issues
